model/app/webapp.go

```diff
@@ -109,7 +109,7 @@ type WebappManifest struct {
 	FromAppsDir bool        `json:"-"` // Used in development
 	Instance    SubDomainer `json:"-"` // Used for JSON-API links
 
-	oldServices Services // Used to diff against when updating the app
+	OldServices Services // Used to diff against when updating the app
```

Comment:
Why do you export this field? At least, if you need to, add `` `json:"-"` `` to this line.

Reply:
We need this field to cast the manifest field in `model/apps/installer.go`

Reply:
But why? It's in the same package, I don't see why it should be exported

Reply:
My bad, my linter was oddly triggered by this